Liquibase lib upgrade to 4.21.1 breaking rollback working earlier

### Search first

- [X] I searched and no similar issues were found

### Description

Hi Team,

Recently we had updated the liquibase library from 4.8.0 to 4.21.1 and we are noticing the below issue when we are executing liquibase.rollback on a single changeset which is present externally

rror updating database: liquibase.exception.ChangeLogParseException: The file tracing_data_15.xml was not found in the configured search path:
More locations can be added with the 'searchPath' parameter.
liquibase.exception.CommandExecutionException: liquibase.exception.ChangeLogParseException: The file tracing_data_15.xml was not found in the configured search path:
More locations can be added with the 'searchPath' parameter.
	at liquibase.command.CommandScope.execute(CommandScope.java:235)
	at liquibase.Liquibase.rollback(Liquibase.java:734)
	at liquibase.Liquibase.rollback(Liquibase.java:717)
	at liquibase.Liquibase.rollback(Liquibase.java:712)
	at liquibase.Liquibase.rollback(Liquibase.java:696)
	at TestProg3.main(TestProg3.java:44)
Caused by: liquibase.exception.ChangeLogParseException: The file tracing_data_15.xml was not found in the configured search path:
More locations can be added with the 'searchPath' parameter.
	at liquibase.parser.core.xml.XMLChangeLogSAXParser.parseToNode(XMLChangeLogSAXParser.java:114)
	at liquibase.parser.core.xml.AbstractChangeLogParser.parse(AbstractChangeLogParser.java:15)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.getDatabaseChangeLog(DatabaseChangelogCommandStep.java:100)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.run(DatabaseChangelogCommandStep.java:77)
	at liquibase.command.CommandScope.execute(CommandScope.java:212)
	... 5 more


he code is simple liquibase.rollback below which used to work with earlier version of library

```
String tracingFilePath = "tracxing_data_15.xml";
            SearchPathResourceAccessor resourceAccessor = new SearchPathResourceAccessor(
                    new DirectoryResourceAccessor(Paths.get("C:\\SIU_1011_RC3\\var\\").toAbsolutePath().toFile()));
            Liquibase liquibase = new Liquibase(tracingFilePath, resourceAccessor, database);
            liquibase.validate();
            liquibase.rollback("tracing_data_15_create", "");
```



### Steps To Reproduce

In windowsplace the tracing_data_15.xml in C:\\SIU_1011_RC3\\var\\ dir

place the lib with below code and dependencies in C:\\SIU_1011_RC3\\ directory
String tracingFilePath = "tracxing_data_15.xml";
            SearchPathResourceAccessor resourceAccessor = new SearchPathResourceAccessor(
                    new DirectoryResourceAccessor(Paths.get("C:\\SIU_1011_RC3\\var\\").toAbsolutePath().toFile()));
            Liquibase liquibase = new Liquibase(tracingFilePath, resourceAccessor, database);
            liquibase.validate();
            liquibase.rollback("tracing_data_15_create", "");

Running the above jar fails with liquibase 4.21.1 library with below exception
More locations can be added with the 'searchPath' parameter.
liquibase.exception.CommandExecutionException: liquibase.exception.ChangeLogParseException: The file tracing_data_15.xml was not found in the configured search path:
More locations can be added with the 'searchPath' parameter.
	at liquibase.command.CommandScope.execute(CommandScope.java:235)
	at liquibase.Liquibase.rollback(Liquibase.java:734)
	at liquibase.Liquibase.rollback(Liquibase.java:717)
	at liquibase.Liquibase.rollback(Liquibase.java:712)
	at liquibase.Liquibase.rollback(Liquibase.java:696)
	at TestProg3.main(TestProg3.java:44)
Caused by: liquibase.exception.ChangeLogParseException: The file tracing_data_15.xml was not found in the configured search path:
More locations can be added with the 'searchPath' parameter.
	at liquibase.parser.core.xml.XMLChangeLogSAXParser.parseToNode(XMLChangeLogSAXParser.java:114)
	at liquibase.parser.core.xml.AbstractChangeLogParser.parse(AbstractChangeLogParser.java:15)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.getDatabaseChangeLog(DatabaseChangelogCommandStep.java:100)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.run(DatabaseChangelogCommandStep.java:77)
	at liquibase.command.CommandScope.execute(CommandScope.java:212)
	... 5 more


The code is simple liquibase.rollback below which used to work with earlier version of library

However the code succeeds with liquibase 4.8.0 library.

### Expected/Desired Behavior

the rollback should succeed ideally

### Liquibase Version

4.21.1

### Database Vendor & Version

mysql 8.0.23

### Liquibase Integration

_No response_

### Liquibase Extensions

liquibase-core

### OS and/or Infrastructure Type/Provider

windows

### Additional Context

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR (Thank you!)

--- a/liquibase-standard/src/main/java/liquibase/Liquibase.java
+++ b/liquibase-standard/src/main/java/liquibase/Liquibase.java
@@ -495,6 +495,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackCountCommandStep.COUNT_ARG, changesToRollback)
                 .addArgumentValue(AbstractRollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .setOutput(new WriterOutputStream(output, GlobalConfiguration.OUTPUT_FILE_ENCODING.getCurrentValue()))
                 .execute();
     }
@@ -536,6 +537,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackCountCommandStep.COUNT_ARG, changesToRollback)
                 .addArgumentValue(AbstractRollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .execute();
     }
     // ---------- End RollbackCount Family of methods
@@ -580,6 +582,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackCommandStep.TAG_ARG, tagToRollBackTo)
                 .addArgumentValue(AbstractRollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .setOutput(new WriterOutputStream(output, GlobalConfiguration.OUTPUT_FILE_ENCODING.getCurrentValue()))
                 .execute();
     }
@@ -631,6 +634,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackCommandStep.TAG_ARG, tagToRollBackTo)
                 .addArgumentValue(RollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .execute();
     }
     // ---------- End Rollback (To Tag) Family of methods
@@ -660,6 +664,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackToDateCommandStep.DATE_ARG, dateToRollBackTo)
                 .addArgumentValue(AbstractRollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .setOutput(new WriterOutputStream(output, GlobalConfiguration.OUTPUT_FILE_ENCODING.getCurrentValue()))
                 .execute();
     }
@@ -705,6 +710,7 @@ public class Liquibase implements AutoCloseable {
                 .addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_ARG, changeExecListener)
                 .addArgumentValue(RollbackToDateCommandStep.DATE_ARG, dateToRollBackTo)
                 .addArgumentValue(AbstractRollbackCommandStep.ROLLBACK_SCRIPT_ARG, rollbackScript)
+                .addArgumentValue(DatabaseChangelogCommandStep.CHANGELOG_PARAMETERS, changeLogParameters)
                 .execute();
 
     }

